.storybook/preview.tsx

```diff
@@ -1,6 +1,6 @@
 import { withThemeByDataAttribute } from '@storybook/addon-themes';
 import type { Preview } from '@storybook/react';
-import '../src/tailwind.css';
+import '../src/globals.css';
```

Comment:
I renamed this file because it contains more styles than just the tailwind stuff. it will contain our color css variables and typography soon too.